templates/primary-secondary/primary-secondary.js

```diff
@@ -758,12 +751,10 @@ class TemplatePrimarySecondary extends RtlMixin(LocalizeCoreElement(LitElement))
 			.d2l-template-primary-secondary-divider:${unsafeCSS(getFocusPseudoClass())} .d2l-template-primary-secondary-divider-handle-left {
 				display: block;
 			}
-			:host(:not([dir="rtl"]):not([secondary-first])) [data-is-expanded] .d2l-template-primary-secondary-divider-handle-left,
```

Comment:
This conditional display was annoying to deal with. It is avoided by using `inset-inline-start`/`inset-inline-end` for the left/right icons, and flipping the icons using `mirror-in-rtl`. We still conditionally hide the icons, but we no longer need to consider RTL when doing it.